src/hb-ot-font.cc

```diff
@@ -44,7 +44,7 @@
 struct hb_ot_face_metrics_accelerator_t
 {
   unsigned int num_metrics;
-  unsigned int num_advances;
+  uint32_t num_advances;
```

Comment:
Meh.  This case, cast would have been nice.  But doesn't matter.  Maybe change `num_metrics` to uint32_t as well?